example/example_source/lib_with_private_kokkos_dependency/bla.cpp

```diff
@@ -0,0 +1,18 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#include <cstdio>
+void print_bla() { printf("Hello From bla\n"); }
```

Comment:
Consistency
```suggestion
#include <iostream>

void print_bla() { std::cout << "Hello from bla \n"); }
```

Reply:
done in https://github.com/kokkos/kokkos/pull/7582/commits/5dd2cc064a66134d5eab3d69e514dacc5f62fb64 and https://github.com/kokkos/kokkos/pull/7582/commits/97081213e09191caa638fbfa87cb56bcaf552cde